src/checkout/ZuoraPaymentForm.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {
+  FC,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import {ZuoraClient} from 'src/types/billing'
+
+// Context
+import {CheckoutContext} from 'src/checkout/context/checkout'
+
+export const ZUORA_SCRIPT_URL =
+  'https://apisandboxstatic.zuora.com/Resources/libs/hosted/1.3.0/zuora-min.js'
```

Comment:
do we want to keep this? Should this be a environment variable?

Reply:
prolly not, but i feel like webpack should be able to handle this somehow